components/EmotionList.js

```diff
@@ -1,6 +1,6 @@
 import styled from "styled-components";
```

Comment:
You have now a lot of code in your EmotionList Component. Please split this up such that the filter display is its own component and is called by your page index.js next to the Emotionlist.